tests/src/Functional/StatusReportTest.php

```diff
@@ -41,6 +41,19 @@ class StatusReportTest extends ApigeeEdgeFunctionalTestBase {
 
   const CANNOT_CONNECT_LONG = 'Cannot connect to Apigee Edge server. You have either given wrong credential details or the Apigee Edge server is unreachable. Visit the Apigee Edge general settings page to get more information.';
 
+  /**
+   * {@inheritdoc}
+   */
+  protected static $mock_api_client_ready = TRUE;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $this->installExtraModules(['apigee_edge_debug']);
```

Comment:
This test had a dependency on `apigee_edge_test` which in turn depends on `apigee_edge_debug`. 

@arlina-espinoza I couldn't really figure out why removing `apigee_edge_debug` makes this test fail.

Reply:
@arshad `apigee_edge_debug` decorates the SDK connector service, and our mock client module alters the service definition by switching an argument (the http client for our mock client), so as it is they are not currently compatible. If you enable `apigee_edge_debug` alongside `apigee_mock_api_client` what happens is that the debug module overwrites the changed parameter and it makes real API calls.
I added a note to the mock client module README that explains why the modules aren't compatible and that refactored tests shouldn't use either `apigee_edge_debug` or `apigee_edge_test`.